docker-compose.yaml

```diff
@@ -52,6 +56,30 @@ services:
     environment:
       BASE_URL: "http://backend:3000/api/v3"
       FACILITY: "SAMPLE-SITE"
+  local-rabbitmq:
+    hostname: local-rabbitmq
```

Comment:
I don't think this is needed

Reply:
That I left in on accident, I think it can be removed without any issues.